PV260_Project/PresentationLayer/Utils/UserInput.cs

```diff
@@ -0,0 +1,27 @@
+namespace PresentationLayer.Utils;
+
+public static class UserInput
+{
+    // Common
```

Comment:
I think it would be prettier to use enums

Reply:
That was the original idea, and when I was searching for C# string enums, I couldn't find anything better than this.
But I just figured out that we can use char enums (somehow I didn't realize that before). We would have to create UserInput enum for each UI class I assume (`EmailUiUserInput`, `DataSetUiUserInput` etc. - or something similar), since e.g. '1' in `EmailUi` means 'Register email', and '1' in `DataSetUi` means 'Load file'. Also, the characters 'q' (`Quit`) and 'b' (`Back`) are used with the same meaning throughout all UI classes and also `App`, so I wonder if it would be better to create something like `CommonUserInput` and use that in every UI class, or just place `Quit` and `Back` inputs in every enum (to only use a single specific enum in each class).

Reply:
I think constants are ok for this usecase.